scripts/run-live-build.sh

```diff
@@ -46,6 +46,13 @@ set -o pipefail
 #
 export APPLIANCE_PASSWORD="${APPLIANCE_PASSWORD:-delphix}"
 
+#
+# Similarly to the appliance credentials above do the same for info
+# needed to sign artifacts.
+#
+export DLPX_KEY_URL="${DLPX_KEY_URL:-https://example.com}"
```

Comment:
What is this default useful for?

Reply:
If we don't actually use the default values for this, and `DLPX_SIGNING_LOGIN`, I'd vote to just not specify defaults.